src/Parser/MistweaverMonk/CombatLogParser.js

```diff
@@ -106,6 +106,11 @@ class CombatLogParser extends MainCombatLogParser {
     petrichorLagniappe: PetrichorLagniappe,
     ovydsWinterWrap: OvydsWinterWrap,
   };
+  
+  damageTaken = 0;
```

Comment:
The reasoning is I need the damage taken without absorbs.  Garg added some math around the damage reduction value of Vers and to limit Vers double dipping on some of our absorbs, I need the value of just the damage taken.  `totalDamageTaken` currently includes absorbs, so I added this for that value.  I can update the current totalDamageTaken method to add that value to the main parser, I just didn't know if other classes / specs would find it useful.

If you are ok, I can implement this in the Core/CombatLogParser as something such as:
`on_toPlayer_damage(event) {
    this.rawDamageTaken += event.amount;
    this.absorbedDamageTaken += event.absorbed || 0;
    this.totalDamageTaken += this.rawDamageTaken + this.absorbedDamageTaken;
  }`